RecoBTag/FeatureTools/plugins/DeepFlavourTagInfoProducer.cc

```diff
@@ -7,6 +7,7 @@
 
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
 #include "FWCore/Utilities/interface/StreamID.h"
+#include "FWCore/Utilities/interface/ESGetToken.h"
```

Comment:
Curious: the updates in this file (which have little to do with clang tidy) were already merged with #30756
How they did show up here once again? Why the original commits from  #30756 do not show up in the github history any more?

Reply:
When I made the PR there was one conflict which came from PR merged touching the same file. I just took the file from #30756 and comited it to resolve the conflict